parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -92,21 +92,44 @@ removeMoby() {
 removeContainerd() {
   apt_get_purge moby-containerd || exit 27
 }
+mobyPkgVersion() {
+  # This gets the version of the package and trims extra bits that the moby packages have (e.g. `+azure`)
```

Comment:
Since this script is part of the cloud-init payload, every byte is precious. So I think the comments here and below need to be omitted, sadly.

Reply:
Ah, interesting.

Reply:
I bet we could do something to strip comments in the generated script.

Reply:
Took comments and extra line breaks out.